docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-jira-issues-for-migration-wave_{context}"]
+= Creating Jira issues for a migration wave
+
```

Comment:
Can you create more than one Jira issue for a single migration wave? If yes, then the title is great. If not, I'd suggest either "Creating Jira issues for migration waves" or "Creating a migration issue for a migration wave". If "not," then you'll also have to adjust the subject-agreement in the passage,  depending on which title option you use. .